start/manage/show.php

```diff
@@ -0,0 +1,74 @@
+<?php 
+require 'header.php';
+
+$id = isset($_GET['id']) ? $_GET['id'] : null;
+
+
+
+
+use Service\Container;
+
+$container = new Container($configuration);
+
+$shipLoader = $container->getShipLoader();
+
+$ship = $shipLoader->findOneById($id);
+
+
+
+//var_dump($ship);die;
+?>
+
+<div class="container">
+    <a href="index.php"> Back to previous page </a>
+
+    <?php 
+    if ($ship === null){
+        echo "Ship not found";
+    } else { ?>
+        <h1> <?php echo $ship->getName(); ?>  </h1>
+
+        <img src="/images/<?php echo $ship->getImage(); ?>" class="img-thumbnail">
```

Comment:
What happens if there isn't an image or it doesn't exist? Should probably handle those cases.